Gems/Atom/Feature/Common/Code/Include/Atom/Feature/CoreLights/LightCommon.h

```diff
@@ -21,25 +21,23 @@ namespace AZ::Render::LightCommon
         return (invRadiusSqaured <= 0.0f) ? 1.0f : Sqrt(1.0f / invRadiusSqaured);
     }
 
-    // Check if a view is being used by a pipeline that has a GPU culling pass.
-    // @param parentScene - Parent Scene associated with a feature processor
-    // @param view - Check if gpu culling passes are part of this view
-    // @param gpuCullingData - Cached Unordered set of Render Pipelines and views that will contain the LightCullingPass which can be used for querying purposes
-    bool HasGPUCulling(
-        RPI::Scene* parentScene,
+    // Check if a view has a pipeline that needs CPU culling (i.e. doesn't have a GPU culling pass).
+    // @param view - The view we are checking against
+    // @param cpuCulledPipelinesPerView - Map of views -> pipelines in that view that need CPU culling (i.e. no GPU culling pass)
+    bool NeedsCPUCulling(
         const RPI::ViewPtr& view,
-        AZStd::unordered_set<AZStd::pair<const RPI::RenderPipeline*, const RPI::View*>>& gpuCullingData);
+        AZStd::unordered_map<const RPI::View*, AZStd::vector<const RPI::RenderPipeline*>>& cpuCulledPipelinesPerView);
```

Comment:
Should be const
```
const AZStd::unordered_map<const RPI::View*, AZStd::vector<const RPI::RenderPipeline*>>& cpuCulledPipelinesPerView
```
Otherwise, it gives the impression that `cpuCulledPipelinesPerView` may get modified.